package-lock.json

```diff
@@ -5,7 +5,8 @@
   "requires": true,
   "packages": {
     "": {
-      "version": "0.3.4",
+      "name": "redis-om",
+      "version": "0.3.5",
```

Comment:
This would need to be part of 0.3.6. Also, update the CHANGELOG to reflect this as well.

Reply:
Technically, this would be a breaking change so let's mark it as 0.4.0 and I'll include this with the  embedded objects release.